ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -1167,6 +1178,183 @@ - (void)getPageCropBox:(NSNumber *)pageNumber resultToken:(FlutterResult)result
+    } else if ([toolMode isEqualToString:PTAnnotationCreateTextUnderlineToolKey]) {
+        toolClass = [PTTextUnderlineCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateTextSquigglyToolKey]) {
+        toolClass = [PTTextSquigglyCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateTextStrikeoutToolKey]) {
+        toolClass = [PTTextStrikeoutCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateFreeTextToolKey]) {
+        toolClass = [PTFreeTextCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateCalloutToolKey]) {
+        toolClass = [PTCalloutCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateSignatureToolKey]) {
+        toolClass = [PTDigitalSignatureTool class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateLineToolKey]) {
+        toolClass = [PTLineCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateArrowToolKey]) {
+        toolClass = [PTArrowCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreatePolylineToolKey]) {
+        toolClass = [PTPolylineCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateStampToolKey]) {
+        toolClass = [PTImageStampCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateRectangleToolKey]) {
+        toolClass = [PTRectangleCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateEllipseToolKey]) {
+        toolClass = [PTEllipseCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreatePolygonToolKey]) {
+        toolClass = [PTPolygonCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreatePolygonCloudToolKey]) {
+        toolClass = [PTCloudCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateDistanceMeasurementToolKey]) {
+        toolClass = [PTRulerCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreatePerimeterMeasurementToolKey]) {
+        toolClass = [PTPerimeterCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateAreaMeasurementToolKey]) {
+        toolClass = [PTAreaCreate class];
+    } else if ([toolMode isEqualToString:PTEraserToolKey]) {
+        toolClass = [PTEraser class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateFreeHighlighterToolKey]) {
+        toolClass = [PTFreeHandHighlightCreate class];
+    } else if ([toolMode isEqualToString:PTAnnotationCreateRubberStampToolKey]) {
+        toolClass = [PTRubberStampCreate class];
+
+    }
+
+    if (toolClass) {
+        PTTool *tool = [docVC.toolManager changeTool:toolClass];
+
+        tool.backToPanToolAfterUse = !self.continuousAnnotationEditing;
+
+        if ([tool isKindOfClass:[PTFreeHandCreate class]]
+            && ![tool isKindOfClass:[PTFreeHandHighlightCreate class]]) {
+            ((PTFreeHandCreate *)tool).multistrokeMode = self.continuousAnnotationEditing;
```

Comment:
Not sure of the rationale here, this should probably be a separate prop (multistroke mode). Unless there is a reason for this of which I'm unaware, I would just leave multistroke on.

Reply:
Copied from RN... (Most of the functions are the same too: https://github.com/PDFTron/pdftron-react-native/blob/master/ios/RNTPTDocumentView.m#L520)

Reply:
@dluco-pdftron what is the rationale for the connection between continuous annotation editing and multistroke?

Reply:
I don't remember the reason why they're connected - likely something to do with there being no multistroke prop at the time. I don't have a preference one way or the other.

Reply:
@dluco-pdftron it seems like a bug to me, where it's likely a misunderstanding of the intention of the continuous annotation editing property.
@Maxwell-Yang-PDFTron I think here you can hard code multistroke to YES, and if a customer complains we can deal with it then. Please test it though to make sure this works as expected.